File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_vb_samples/App_Code/VisualBasicWebApp/CourseExample.vb
Imports Microsoft.VisualBasic

Public Class CourseExample
    Public Sub New(ByVal _title As String, ByVal _description As String, ByVal _URL As String)
        Me._title = _title
        Me._description = _description
        Me._URL = _URL
    End Sub
    Public Property URL() As String
        Get
            Return _URL
        End Get
        Set(ByVal value As String)
            _URL = value
        End Set
    End Property
    Public Property Description() As String
        Get
            Return _description
        End Get
        Set(ByVal value As String)
            _description = value
        End Set
    End Property
    Public Property Title() As String
        Get
            Return _title
        End Get
        Set(ByVal value As String)
            _title = value
        End Set
    End Property
    Private _URL As String
    Private _description As String
    Private _title As String
End Class

File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_cs_samples/App_Code/CourseExample.cs
using System;

namespace aspxsamples {
	public class CourseExample {
		private readonly string url;
		private readonly string description;
		private readonly string title;

		public CourseExample(string title, string description, string url) {
			this.title = title;
			this.description = description;
			this.url = url;
		}
		public string Url {
			get { return url; }
		}
		public string Description {
			get { return description; }
		}
		public string Title {
			get { return title; }
		}
	}
}